cea/interfaces/arcgis/CityEnergyAnalyst.py

```diff
@@ -93,7 +93,7 @@ def __init__(self):
         self.category = 'Optimization'
         self.canRunInBackground = False
 
-class DecentralizedBuildings(CeaTool):
+class Decentralized(CeaTool):
```

Comment:
No. This just makes it worse. "Decentralized" could mean anything. Decentralized what?